modules/service/src/main/resources/logback.xml

```diff
@@ -10,14 +10,14 @@
   <logger name="org.http4s.blazecore.IdleTimeoutStage" level="OFF" />
   <logger name="org.http4s.blazecore.websocket.Http4sWSStage" level="OFF" />
 
-  <logger name="lucuma-odb" level="ALL" />
+  <logger name="lucuma-odb" level="WARN" />
```

Comment:
Is this the level we want for this? I think it was INFO before Rob made it more verbose. I have no particular insight into what this should be, but maybe someone else does?